pkg/sdk/file_format_test.go

```diff
@@ -85,14 +75,10 @@ func TestFileFormatsCreate(t *testing.T) {
 				JSONStripOuterArray:          Bool(true),
 				JSONStripNullValues:          Bool(true),
 				JSONReplaceInvalidCharacters: Bool(true),
-				JSONIgnoreUTF8Errors:         Bool(true),
```

Comment:
why was it removed ?

Reply:
Because this is not valid configuration (cant be used with JSONReplaceInvalidCharacters)